### PR TITLE
Header and footer

### PR DESCRIPTION
**Pull request by SulimanJoya**
1. Added header
2. Added Footer
3. style header and footer

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -2,5 +2,57 @@
...
+.d-flex {
+  display: flex;
+}
+
+.navbar {
+  background-color: #444;
+  color: #fff;
+  padding: 0 15%;
+  font-weight: 600;
+}
```
Comment (addisalemseifu) [inline src/index.css:23]: -Please make the `.navbar` `display: fixed` so that it will stick to the top of the page.

**Review by addisalemseifu — CHANGES_REQUESTED** [review]
229514525-8e23ea47-0e1c-42d0-aaa7-9543c9dd5dee
-Dear Suliman you have done great so far there are just a few things you need to improve.

File: src/index.css (hunk 1 above)
Comment (SulimanJoya) [reply src/index.css:23]: Thanks for your valuable feedback @addisalemseifu  as you said make the nav as display fixed I thank it is not a part of the requirement so please check the code once more if there is something to change let me know. thanks

**Review by addisalemseifu — APPROVED** [review]
Hi @addisalemseifu and @amon-cofie 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Use jest framework for testing ✔️ 
All test passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**